### PR TITLE
Bump dispatch to version 0.11.3

### PR DESCRIPTION
**Pull request by DustinChaloupka**
This bumps the dispatch version to `0.11.3`. I was unsure if there were specific tests to be added for each specific version of dispatch instead of just the tests handling whatever the default dispatch version were, but they all passed.

### CODE REVIEW

**Comment by eed3si9n** [comment]
Currently the announced "latest" is 0.11.2 on http://dispatch.databinder.net/Dispatch.html. See also https://groups.google.com/d/msg/dispatch-scala/iy12POc81LM/udzLH14ajcMJ

**Comment by eed3si9n** [comment]
Also what's the difference between httpclients_dispatch0111.scala.template and httpclients_dispatch0113.scala.template?

**Comment by DustinChaloupka** [comment]
The only differences are the changes in the method names.

`setConnectionTimeoutInMs` -> `setConnectTimeout`

Also did not know that the `0.11.3` version not "officially" released, but was working with something that is using it so figured I would open this PR. Feel free to close it if will not be used. :)

File: project/dependencies.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ object Dependencies {
 
   val scopt = "com.github.scopt" %% "scopt" % "3.2.0"
   val log4j = "log4j" % "log4j" % "1.2.17"
-  val defaultDispatchVersion = "0.11.2"
+  val defaultDispatchVersion = "0.11.3"
```
Comment (eed3si9n) [inline project/dependencies.scala]: I think this pull request overall is useful if source change was required due to dispatch/reboot#116, but could we keep the defaultDispatchVersion to 0.11.2?

Comment (DustinChaloupka) [inline project/dependencies.scala]: Changed.

**Comment by eed3si9n** [comment]
Thanks for the contribution.

**Pull request merged**